packages/shared/src/graphql/squads.ts

```diff
@@ -189,6 +199,27 @@ export const SQUAD_HANDE_AVAILABILITY_QUERY = gql`
   }
 `;
 
+export const SQUAD_MEMBERSHIPS_QUERY = gql`
+  query Source {
+    mySourceMemberships {
+      sources: edges {
```

Comment:
I think `members` is much more accurate.
```suggestion
      members: edges {
```

Reply:
I think it returns sources, not members @sshanzel it's just the sources this user is part of

Reply:
I think it is members, where it has a `source` property.

Reply:
I think this is members, where it has the source property.

Reply:
Just confirmed, it is indeed members.

![image](https://user-images.githubusercontent.com/13744167/214502353-cb14cf29-dc34-4c4c-8011-41c4c5bbcec0.png)

Reply:
Oh, not sure I like the name `mySourceMemberships` that much in that case 😅

Reply:
It returns `SourceMember` objects hence `SourceMemberships`. This was the logic I followed.
Usually the best practice would be to return the complete object which in this case is `SourceMember` rather than a child directly. Because `SourceMember` contains relevant data